tests/packages/test-metadata/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[build-system]
+requires = []
+build-backend = 'backend'
+backend-path = ['.']
+
+[project]
```

Comment:
By not making this tool.build you're technically violating pep-518... :+1:

Reply:
These are PEP 621 metadata.